data-prepper-plugins/cloudwatch-logs/src/test/java/org/opensearch/dataprepper/plugins/sink/buffer/InMemoryBufferTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.sink.buffer;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.event.JacksonEvent;
+import org.opensearch.dataprepper.model.record.Record;
```

Comment:
Record is deprecated.

Reply:
Record is deprecated, but still used as part of Data Prepper. I mainly used it as the other plugins use it as standard for output method calls.
@kkondaka @dinujoh What instance should be used in its stead or will this be added later on?

Reply:
It's OK to use Record. It is my understanding It is deprecated in using the metadata but not entirely. @dlvenable, why is Record marked deprecared?